JsonSubTypes.Tests/DiscriminatorLocationTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace JsonSubTypes.Tests
+{
+
+    class SimpleBase
+    {
+        public String Name { get; set; } 
+    }
+
+    class SimpleChildA : SimpleBase
+    {
+        public int Age { get; set; }
+    }
+
+    class SimpleChildB : SimpleBase
+    {
+        public int Height { get; set; }
+    }
+
+    [TestFixture]
+    public class DiscriminatorLocationTests
+    {
+        [Test]
+        public void CheckFirst()
```

Comment:
Thanks for the tests ! 

It might also be a good idea to add test case that describe the default behaviour (with and without the parameter)

Reply:
@manuc66 added two such test cases

Reply:
Thanks